probe-rs/src/architecture/arm/sequences/stm32h7.rs

```diff
@@ -149,7 +151,7 @@ impl ArmDebugSequence for Stm32h7 {
         _default_ap: MemoryAp,
         _permissions: &crate::Permissions,
     ) -> Result<(), ArmError> {
-        // Power up the debug components through AP2, which is the defualt AP debug port.
+        // Power up the debug components through AP2, which is the default AP debug port.
         let ap = MemoryAp::new(ApAddress {
```

Comment:
This is not your code but do you know why we do not use `default_ap` here?

Reply:
Same as in debug_core_stop, AP 2 is the "debug AP" in a STM32H7 according to documentation.